### PR TITLE
Eri9600/project sample

### PR DESCRIPTION
**Pull request by eri9600**
Sample using Geometry Engine to project a a point in another spatial reference.
shown here: https://devtopia.esri.com/runtime/common-samples/issues/196

### CODE REVIEW

**Comment by eri9600** [comment]
@TADraeseke  this sample should also go into develop. I am using arcgis version '100.3.0-2066'.

File: java/project/build.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        applicationId "com.esri.arcgisruntime.sample.displaymap"
```
Comment (TADraeseke) [inline java/project/build.gradle]: .project

File: java/project/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Project
+Being able to project between spatial references is fundamental to a GIS.
+An example of when you would need to re-project data is if you had data
```
Comment (TADraeseke) [inline java/project/README.md]: Let text wrap itself in READMEs (don't put carrier returns inbetween lines)

File: java/project/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+in two different spatial references, but wanted to perform an intersect 
+analysis with the `GeometryEngine::intersect` function. This function 
+takes two geometries as parameters, and both geometries must be in the 
+same spatial reference. If they are not, you could first use `GeometryEngine::project` 
+to convert the geometries so they match.
+
+![Project App](project.png)
+
+## How to use the sample
+Click anywhere on the map. A callout will display the clicked location's coordinate in
```
Comment (TADraeseke) [inline java/project/README.md]: remove carrier return

File: java/project/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+to convert the geometries so they match.
+
+![Project App](project.png)
+
+## How to use the sample
+Click anywhere on the map. A callout will display the clicked location's coordinate in 
+the original (basemap's) spatial reference and in the projected spatial reference.
+
+## How it works
+To project a geometry to another spatial reference:
```
Comment (TADraeseke) [inline java/project/README.md]: Can just make this a sentence without number if there's only one step.

File: java/project/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+1. Call the static method, `GeometryEngine.project`, passing in the original 
+`Geometry` and a `SpatialReference` to project to.
+
+## Relevant API  
+- GeometryEngine
```
Comment (TADraeseke) [inline java/project/README.md]: * instead of - (I think Thad's script relies on this)

File: java/project/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- Point
+- SpatialReference
+
+## Tags
```
Comment (TADraeseke) [inline java/project/README.md]: ' #### Tags'

File: java/project/src/main/java/com/esri/arcgisruntime/sample/project/MainActivity.java [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+package com.esri.arcgisruntime.sample.project;
+
+import java.text.DecimalFormat;
+
+import android.graphics.Color;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.MotionEvent;
+import android.widget.TextView;
+import com.esri.arcgisruntime.geometry.Envelope;
```
Comment (TADraeseke) [inline java/project/src/main/java/com/esri/arcgisruntime/sample/project/MainActivity.java]: carrier return between android and runtime imports

File: java/project/src/main/java/com/esri/arcgisruntime/sample/project/MainActivity.java [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+        calloutContent.setTextColor(Color.BLACK);
+        calloutContent.setText(String.format("Coordinates\nOriginal: %s, %s\nProjected: %s, %s", ox, oy, px, py));
+        // create callout
+        final Callout callout = mMapView.getCallout();
+        callout.setLocation(originalPoint);
+        callout.setContent(calloutContent);
+        callout.show();
+        return true;
+      }
+
```
Comment (TADraeseke) [inline java/project/src/main/java/com/esri/arcgisruntime/sample/project/MainActivity.java]: remove whitespace

**Comment by eri9600** [comment]
@TADraeseke made all the suggested changes. 
Thanks!

File: java/project/README.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+To project a geometry to another spatial reference:
+* Call the static method, `GeometryEngine.project`, passing in the original `Geometry` and a `SpatialReference` to project to.
+
+## Relevant API  
+* GeometryEngine
+* Point
+* SpatialReference
+
+#### Tags
+GeometryEngine
```
Comment (TADraeseke) [inline java/project/README.md]: Make first tag 'Edit and Manage Data' so it gets picked up by the SampleViewer script.

**Comment by eri9600** [comment]
@TADraeseke I added the extra Tag. Merging now.

**Pull request merged**